src/fluent-bit.c

```diff
@@ -842,7 +842,8 @@ int main(int argc, char **argv)
         flb_utils_error(FLB_ERR_OUTPUT_UNDEF);
     }
 
-    if (config->verbose == FLB_TRUE) {
+    /* debug or warning */
+    if (config->verbose > 3) {
```

Comment:
I'd suggest for clarity:  `if (config->verbose >= FLB_LOG_DEBUG)` which is functionally equivalent.